Panic occurs: index out of range

## Environment

* macOS Big Sur

```sh
$ golangci-lint version
golangci-lint has version 1.41.1 built from a207480 on 2021-06-19T21:15:29Z
```

## Code

sample.go

```go
package main

import (
	"fmt"

	"github.com/rs/xid"
)

func main() {
	guid := xid.New()
	fmt.Printf("%s: %d\n", guid.String(), guid.Counter())
}
```

## Run

```sh
$ golangci-lint run sample.go --disable-all --enable promlinter

ERRO [runner] Panic: promlinter: package "main" (isInitialPkg: true, needAnalyzeSource: true): runtime error: index out of range [0] with length 0: goroutine 107 [running]:
runtime/debug.Stack(0x4e48f29, 0x3c, 0xc000df2cb0)
        runtime/debug/stack.go:24 +0x9f
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyzeSafe.func1(0xc0002e86c0)
        github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner_action.go:101 +0x1be
panic(0x4d12840, 0xc00003cc00)
        runtime/panic.go:965 +0x1b9
github.com/yeya24/promlinter.(*visitor).parseCallerExpr(0xc000de7800, 0xc000de7700, 0x4f98a00, 0x4fa3e30)
        github.com/yeya24/promlinter@v0.1.0/promlinter.go:253 +0xaaf
github.com/yeya24/promlinter.(*visitor).Visit(0xc000de7800, 0x4fa37a0, 0xc000de7700, 0x4f98a40, 0xc000de7800)
        github.com/yeya24/promlinter@v0.1.0/promlinter.go:167 +0xde
go/ast.Walk(0x4f98a40, 0xc000de7800, 0x4fa37a0, 0xc000de7700)
        go/ast/walk.go:52 +0x63
go/ast.walkExprList(0x4f98a40, 0xc000de7800, 0xc000de7740, 0x3, 0x4)
        go/ast/walk.go:26 +0x9e
go/ast.Walk(0x4f98a40, 0xc000de7800, 0x4fa37a0, 0xc000de7780)
        go/ast/walk.go:137 +0xfc5
go/ast.Walk(0x4f98a40, 0xc000de7800, 0x4fa39f8, 0xc000e6c570)
        go/ast/walk.go:196 +0x1a05
go/ast.walkStmtList(0x4f98a40, 0xc000de7800, 0xc000e8e4c0, 0x2, 0x2)
        go/ast/walk.go:32 +0x9e
go/ast.Walk(0x4f98a40, 0xc000de7800, 0x4fa3750, 0xc000e90810)
        go/ast/walk.go:224 +0x1968
go/ast.Walk(0x4f98a40, 0xc000de7800, 0x4fa3a98, 0xc000de77c0)
        go/ast/walk.go:275 +0x6aa
go/ast.walkStmtList(0x4f98a40, 0xc000de7800, 0xc000e6c580, 0x1, 0x1)
        go/ast/walk.go:32 +0x9e
go/ast.Walk(0x4f98a40, 0xc000de7800, 0x4fa3750, 0xc000e90840)
        go/ast/walk.go:224 +0x1968
go/ast.Walk(0x4f98a40, 0xc000de7800, 0x4fa3ac0, 0xc000e90870)
        go/ast/walk.go:344 +0xce6
go/ast.walkDeclList(0x4f98a40, 0xc000de7800, 0xc000e8e4e0, 0x2, 0x2)
        go/ast/walk.go:38 +0x9e
go/ast.Walk(0x4f98a40, 0xc000de7800, 0x4fa3a70, 0xc000de3480)
        go/ast/walk.go:353 +0x2446
github.com/yeya24/promlinter.RunLint(0xc0001a1ac0, 0xc000010b88, 0x1, 0x1, 0xc000e96200, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        github.com/yeya24/promlinter@v0.1.0/promlinter.go:125 +0x145
github.com/golangci/golangci-lint/pkg/golinters.NewPromlinter.func1.1(0xc000de5860, 0x112d2f23, 0x572d300, 0xc0002e8798, 0x2)
        github.com/golangci/golangci-lint/pkg/golinters/promlinter.go:34 +0x9a
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyze(0xc0002e86c0)
        github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner_action.go:187 +0x9f2
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyzeSafe.func2()
        github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner_action.go:105 +0x2a
github.com/golangci/golangci-lint/pkg/timeutils.(*Stopwatch).TrackStage(0xc000e48780, 0x4dcd136, 0xa, 0xc00061d770)
        github.com/golangci/golangci-lint/pkg/timeutils/stopwatch.go:111 +0x4d
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyzeSafe(0xc0002e86c0)
        github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner_action.go:104 +0x91
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*loadingPackage).analyze.func2(0xc000e80250, 0xc0002e86c0)
        github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner_loadingpackage.go:80 +0x65
created by github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*loadingPackage).analyze
        github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner_loadingpackage.go:75 +0x316 
WARN [runner] Can't run linter promlinter: promlinter: promlinter: package "main" (isInitialPkg: true, needAnalyzeSource: true): runtime error: index out of range [0] with length 0 

```

## go get

I tried to install and use `promlinter` with go get, but I get the same error.

```sh
go get github.com/yeya24/promlinter/cmd/promlinter
```

```
$ promlinter lint sample.go

panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/yeya24/promlinter.(*visitor).parseCallerExpr(0xc0000c8280, 0xc0000c81c0, 0x14d7700, 0x14da378)
        /Users/jnst/go/pkg/mod/github.com/yeya24/promlinter@v0.1.0/promlinter.go:253 +0xaaf
github.com/yeya24/promlinter.(*visitor).Visit(0xc0000c8280, 0x14d9e00, 0xc0000c81c0, 0x14d77a0, 0xc0000c8280)
        /Users/jnst/go/pkg/mod/github.com/yeya24/promlinter@v0.1.0/promlinter.go:167 +0xde
go/ast.Walk(0x14d77a0, 0xc0000c8280, 0x14d9e00, 0xc0000c81c0)
        /Users/jnst/.goenv/versions/1.16.2/src/go/ast/walk.go:52 +0x63
go/ast.walkExprList(0x14d77a0, 0xc0000c8280, 0xc0000c8200, 0x3, 0x4)
        /Users/jnst/.goenv/versions/1.16.2/src/go/ast/walk.go:26 +0x9e
go/ast.Walk(0x14d77a0, 0xc0000c8280, 0x14d9e00, 0xc0000c8240)
        /Users/jnst/.goenv/versions/1.16.2/src/go/ast/walk.go:137 +0xfc5
go/ast.Walk(0x14d77a0, 0xc0000c8280, 0x14d9fb8, 0xc0000984b0)
        /Users/jnst/.goenv/versions/1.16.2/src/go/ast/walk.go:196 +0x1a05
go/ast.walkStmtList(0x14d77a0, 0xc0000c8280, 0xc00009a380, 0x2, 0x2)
        /Users/jnst/.goenv/versions/1.16.2/src/go/ast/walk.go:32 +0x9e
go/ast.Walk(0x14d77a0, 0xc0000c8280, 0x14d9db0, 0xc00009e7b0)
        /Users/jnst/.goenv/versions/1.16.2/src/go/ast/walk.go:224 +0x1968
go/ast.Walk(0x14d77a0, 0xc0000c8280, 0x14da080, 0xc00009e7e0)
        /Users/jnst/.goenv/versions/1.16.2/src/go/ast/walk.go:344 +0xce6
go/ast.walkDeclList(0x14d77a0, 0xc0000c8280, 0xc00009a3a0, 0x2, 0x2)
        /Users/jnst/.goenv/versions/1.16.2/src/go/ast/walk.go:38 +0x9e
go/ast.Walk(0x14d77a0, 0xc0000c8280, 0x14da030, 0xc0000a4180)
        /Users/jnst/.goenv/versions/1.16.2/src/go/ast/walk.go:353 +0x2446
github.com/yeya24/promlinter.RunLint(0xc0000c8080, 0xc00009c120, 0x1, 0x1, 0xc00009c100, 0x0, 0x0, 0x0, 0x0, 0xc000159df0, ...)
        /Users/jnst/go/pkg/mod/github.com/yeya24/promlinter@v0.1.0/promlinter.go:125 +0x145
main.main()
        /Users/jnst/go/pkg/mod/github.com/yeya24/promlinter@v0.1.0/cmd/promlinter/main.go:80 +0xe10

```

--- a/promlinter.go
+++ b/promlinter.go
@@ -250,6 +250,10 @@ func (v *visitor) parseCallerExpr(call *ast.CallExpr) ast.Visitor {
 		return v
 	}
 
+	if len(call.Args) == 0 {
+		return v
+	}
+
 	return v.parseOpts(call.Args[0], metricType)
 }
 

